apps/ui/components/reservation-unit/ReservationInfoContainer.tsx

```diff
@@ -54,61 +49,65 @@ const ReservationInfoContainer = ({
         {t("reservationCalendar:reservationInfo")}
       </Subheading>
       <Content data-testid="reservation-unit__reservation-info">
-        {reservationUnitIsReservable && (
-          <p>
-            {maxDaysBefore > 0 && minDaysBefore > 0 && (
-              <Trans
-                i18nKey="reservationUnit:reservationInfo1-1"
-                defaults="Voit tehdä varauksen <strong>aikaisintaan {{reservationsMaxDaysBefore}} {{unit}}</strong> ja <bold>viimeistään {{reservationsMinDaysBefore}} päivää etukäteen</bold>."
-                values={{
-                  reservationsMaxDaysBefore: daysByMonths.find(
-                    (n) => n.value === reservationUnit.reservationsMaxDaysBefore
-                  )?.label,
-                  unit: t(
-                    `reservationUnit:reservationInfo1-${
-                      reservationUnit.reservationsMaxDaysBefore === 14
-                        ? "weeks"
-                        : "months"
-                    }`
-                  ),
-                  reservationsMinDaysBefore:
-                    reservationUnit.reservationsMinDaysBefore,
-                }}
-                components={{ bold: <strong /> }}
-              />
-            )}
-            {maxDaysBefore > 0 && minDaysBefore === 0 && (
-              <Trans
-                i18nKey="reservationUnit:reservationInfo1-2"
-                defaults="Voit tehdä varauksen <bold>aikaisintaan {{reservationsMaxDaysBefore}} {{unit}} etukäteen</bold>."
-                values={{
-                  reservationsMaxDaysBefore: daysByMonths.find(
-                    (n) => n.value === reservationUnit.reservationsMaxDaysBefore
-                  )?.label,
-                  unit: t(
-                    `reservationUnit:reservationInfo1-${
-                      reservationUnit.reservationsMaxDaysBefore === 14
-                        ? "weeks"
-                        : "months"
-                    }`
-                  ),
-                }}
-                components={{ bold: <strong /> }}
-              />
-            )}
-            {maxDaysBefore === 0 && minDaysBefore > 0 && (
-              <Trans
-                i18nKey="reservationUnit:reservationInfo1-3"
-                defaults="Voit tehdä varauksen <bold>viimeistään {{reservationsMinDaysBefore}} päivää etukäteen</bold>."
-                values={{
-                  reservationsMinDaysBefore:
-                    reservationUnit.reservationsMinDaysBefore,
-                }}
-                components={{ bold: <strong /> }}
-              />
-            )}
-          </p>
-        )}
+        {reservationUnitIsReservable &&
+          (reservationUnit.reservationsMaxDaysBefore != null ||
+            reservationUnit.reservationsMinDaysBefore != null) && (
```

Comment:
Is this some sonar lint thing? I don't like it. Creates extra nesting level for no reason in the JSX.